project/profiles/capacity4more/themes/c4m/kapablo/theme/preprocess.inc

```diff
@@ -51,6 +50,10 @@ function kapablo_preprocess_message(&$variables) {
  * Implements theme_preprocess_block().
  */
 function kapablo_preprocess_block(&$variables) {
+  if (isset($variables['elements']['#block']->info) && $variables['elements']['#block']->info == 'OG Menu : single') {
```

Comment:
@ybaras 
`$variables['elements']['#block']->info == 'OG Menu : single'`
this one looks weird a bit ...